bc_obps/registration/migrations/0012_facility_alter_operation_table_comment_and_more.py

```diff
@@ -0,0 +1,307 @@
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('registration', '0011_historicaloperation_address_operation_address'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Facility',
+            fields=[
+                ('created_at', models.DateTimeField(auto_now_add=True, null=True)),
+                ('updated_at', models.DateTimeField(blank=True, null=True)),
+                ('archived_at', models.DateTimeField(blank=True, null=True)),
+                ('name', models.CharField(db_comment="An operation or facility's name", max_length=1000)),
+                ('type', models.CharField(db_comment="An operation or facility's type", max_length=1000)),
+                (
+                    'id',
+                    models.UUIDField(
+                        db_comment='Primary key to identify the facility',
+                        default=uuid.uuid4,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name='ID',
+                    ),
+                ),
+                ('new_entrant', models.BooleanField(db_comment='Whether or not the facility is a new entrant')),
+                (
+                    'address',
+                    models.ForeignKey(
+                        blank=True,
+                        db_comment='The address of the operation or facility',
+                        null=True,
+                        on_delete=django.db.models.deletion.DO_NOTHING,
+                        related_name='%(class)s_address',
+                        to='registration.address',
+                    ),
+                ),
+                (
+                    'archived_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_archived',
+                        to='registration.user',
+                    ),
+                ),
+                (
+                    'created_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_created',
+                        to='registration.user',
+                    ),
+                ),
+                (
+                    'updated_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_updated',
+                        to='registration.user',
+                    ),
+                ),
+            ],
+            options={
+                'db_table': 'erc"."facility',
+                'db_table_comment': 'Facilities ',
```

Comment:
This table comment needs more information
```suggestion
                'db_table_comment': 'Contains data on facilities that emit carbon emissions and must report them to Clean Growth. A linear facility operation is made up of several different facilities whereas a single facility operation has only one facility. In the case of a single facility operation, much of the data in this table will overlap with the parent record in the operation table',
```